### PR TITLE
SEARCH-2624 add an argument to identify a phrase query

### PR DESCRIPTION
**Pull request by jesty**
This PR will be used by https://github.com/Alfresco/alfresco-enterprise-repo/pull/180

### CODE REVIEW

File: data-model/src/main/java/org/alfresco/repo/search/impl/querymodel/impl/functions/FTSPhrase.java [hunk 1]
```diff
@@ -47,6 +47,8 @@
     public final static String ARG_PHRASE = "Phrase";
 
     public final static String ARG_PROPERTY = "Property";
+
+    public final static String ARG_IS_PHRASE = "IsPhrase";
```
Comment (killerboot) [inline data-model/src/main/java/org/alfresco/repo/search/impl/querymodel/impl/functions/FTSPhrase.java:51]: This name clashes with ARG_PHRASE. Please change it or add a clear JavaDoc explaining what it means.

**Review by killerboot — CHANGES_REQUESTED** [review]
Please add tests for your change.

File: data-model/src/main/java/org/alfresco/repo/search/impl/querymodel/impl/functions/FTSPhrase.java (hunk 1 above)
Comment (jesty) [reply data-model/src/main/java/org/alfresco/repo/search/impl/querymodel/impl/functions/FTSPhrase.java:51]: What do you think about ARG_HAS_QUOTATION_MARKS? Meanwhile, I will add a JavaDoc.

Comment (killerboot) [reply data-model/src/main/java/org/alfresco/repo/search/impl/querymodel/impl/functions/FTSPhrase.java:55]: I'm not sure, perhaps you can use a short version ARG_HAS_QUOTES.

**Comment by jesty** [comment]
> Please add tests for your change.

@killerboot I didn't found any class where to add tests, so I created a new one: FTSQueryParserTest

**Comment by jesty** [comment]
I close the PR because the change is not more required.

**Pull request closed without merging**